Makefile

```diff
@@ -1,29 +1,23 @@
-dirs = dku_config # Add here new libs separated by a whitespace if needed
+modules = dku_config # Add here new libs separated by a whitespace if needed
 
 test-one:
-	@echo "[START] Running unit tests on ${f}..."
+	@echo "[START] Running unit tests on ${module}..."
 	@( \
 		rm -rf env; \
 		python3 -m venv env/; \
 		source env/bin/activate; \
 		pip3 install --upgrade pip; \
-		pip install --no-cache-dir -r tests/${f}/requirements.txt; \
-		pip install --no-cache-dir -r dkulib/${f}/requirements.txt; \
-		pytest -o junit_family=xunit2 --junitxml=unit.xml tests/${f} || true; \
+		pip3 install --no-cache-dir -r dkulib/${module}/requirements.txt -r tests/requirements.txt; \
```

Comment:
Jenkins executes whatever you tell him to, so that's up to us to decide. 

Ideally, I would rather have only one requirements.txt for all tests, so that we keep module-specific requirements in dkulib. But that's just me - if you have identified cases where you may need different requirements for each test, happy to change that.

Reply:
@HenriChabert is that OK with you? Should we resolve?

Reply:
I guess you are right, I can't identify any use case whee it would be useful to have separate libs. Let' s leave it that way.